### DISCUSSION TITLE
Multiple levels of model inheritance does not work

### DISCUSSION THREAD

**Discussion by flashgorman-chorus** [fastapi/fastapi#7353] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Example

Here's a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) with my use case:

```Python
from fastapi import FastAPI
from pydantic import Field, BaseModel

app = FastAPI()


class ConversationAttributesForCreate(BaseModel):
    name: str = Field(None, title="The name of the conversation.")
    description: Optional[str] = Field(
        None, title="The description of the conversation", max_length=200
    )


class ConversationAttributes(ConversationAttributesForCreate):
    date_created: str


class Conversation(BaseModel):
    attributes = ConversationAttributes
    id: str
    type: str


@app.put("/api/v1/conversations/{id}", tags=["Conversations"])
def update_a_conversation(id: str, conversation: Conversation):
    print(f"update conversation {id}: {conversation}")
```

### Description

After defining the "update a conversation API" as shown above, the resulting schema (by way of the following example) looks like this:

```
{
  "id": "string",
  "type": "string"
}
```

Where are the attributes?

**Comment by old-syniex** [comment]
Hey,
Try to use it as a type. 
`class Conversation(BaseModel):
    attributes: ConversationAttributes
    id: str
    type: str`

**Comment by MacMacky** [comment, marked as answer]
Hi @flashgorman-chorus , I think you should use the type not the value.

```py
class Conversation(BaseModel):
    attributes: ConversationAttributes
    id: str
    type: str
```

**Comment by flashgorman-chorus** [comment]
Oh, man!  I feel stupid!  Nice response!

**Comment by tiangolo** [comment]
Thanks for the help here everyone! :clap: :bow: 

Thanks for reporting back and closing the issue :+1:
